ITKIOScanco_UnitConversion.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import numpy as np
</pre><p>Before importing, can we first have a cell that installs all dependencies so the notebook is more reproducible? See, e.g. https://github.com/KitwareMedical/ITKIOScanco/blob/master/examples/ReadISQ.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/ITKIOScanco/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>